### PR TITLE
AM-12443: 0SS v1.2.0 release - Support for private clusters

### PR DESCRIPTION
**Pull request by uma-kt**
Staging link:

Release notes: https://staging-docs.avesha.io/documentation/open-source/1.2.0/release-notes/release-notes-for-kubeslice-oss-1.2.0

Kubeslice components
https://staging-docs.avesha.io/documentation/open-source/1.2.0/overview/components#slice-gateway-edge

Key Features
https://staging-docs.avesha.io/documentation/open-source/1.2.0/overview/features#connectivity-to-clusters-in-private-vpcs

YAML description
https://staging-docs.avesha.io/documentation/open-source/1.2.0/install-kubeslice/yaml/slice-operations/slice-operations-slice-creation#slice-creation

Feature description
https://staging-docs.avesha.io/documentation/open-source/1.2.0/install-kubeslice/yaml/slice-operations/slice-operations-slice-creation#inter-cluster-connectivity-in-kubeslice

### CODE REVIEW

File: versioned_docs/version-1.2.0/install-kubeslice/yaml/slice-operations/slice-operations-slice-creation.mdx [hunk 1]
```diff
@@ -0,0 +1,916 @@
...
+ sliceGatewayProvider:
+    sliceGatewayType: OpenVPN
+    sliceGatewayServiceType:
+      - cluster: <registered-cluster-name-1>
+        type: LoadBalancer   # By default, the gateway service type is NodePort.
+        protocol: TCP        # By default. the gateway protocol is UDP.
+      - cluster: <registered-cluster-name-2>
+        type: LoadBalancer   # By default, the gateway service type is NodePort.
+        protocol: TCP        # By default, the gateway protocol is UDP.   
+```
```
Comment (veenamj-avesha) [inline versioned_docs/version-1.2.0/install-kubeslice/yaml/slice-operations/slice-operations-slice-creation.mdx:288]: This was added as a sub-heading in EE v1.5.0: https://docs.avesha.io/documentation/enterprise/1.5.0/install-kubeslice/yaml/slice-operations/slice-operations-slice-creation#change-the-gateway-provider-for-inter-cluster-connectivity. 

They are in the EE v1.5.0 source docs. Can you copy that content here?

File: versioned_docs/version-1.2.0/release-notes/release-notes-for-kubeslice-oss-1.2.0.mdx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Release Notes for KubeSlice OSS 1.2.0
+
+*Release date: 9th Jan 2024*
```
Comment (veenamj-avesha) [inline versioned_docs/version-1.2.0/release-notes/release-notes-for-kubeslice-oss-1.2.0.mdx:3]: This date needs to be changed to the actual release date.

File: versioned_docs/version-1.2.0/install-kubeslice/yaml/slice-operations/slice-operations-slice-creation.mdx (hunk 1 above)
Comment (uma-kt) [reply versioned_docs/version-1.2.0/install-kubeslice/yaml/slice-operations/slice-operations-slice-creation.mdx:288]: done

File: versioned_docs/version-1.2.0/overview/components.mdx [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+* [Slice Operator](#slice-operator)
+* [Slice VPN Gateways](#slice-vpn-gateways)
+* [Slice Router](#slice-router)
+* [Slice Istio Components](#slice-istio-components)
+* [KubeSlice DNS](#kubeslice-dns)
+* [Network Service Mesh Control and Data Plane](#network-service-mesh-control-and-data-plane)
+* [NSM Kernel Forwarder](#nsm-kernel-forwarder)
+* [Spire Server and Agents](#spire-server-and-agents)
+* [NetOps](#netops)
+* [Slice Gateway Edge](#slice-gateway-edge)
```
[the remark this answers is not in the document]
Comment (uma-kt) [reply versioned_docs/version-1.2.0/overview/components.mdx]: done

File: versioned_docs/version-1.2.0/release-notes/release-notes-for-kubeslice-oss-1.2.0.mdx (hunk 2 above)
Comment (uma-kt) [reply versioned_docs/version-1.2.0/release-notes/release-notes-for-kubeslice-oss-1.2.0.mdx]: done

**Pull request merged**